QgisModelBaker/tests/test_import.py

```diff
@@ -178,6 +179,85 @@ def test_import_geopackage(self):
         cursor.close()
         conn.close()
 
+    def test_import_mssql(self):
+
+        # Schema Import
+        importer = iliimporter.Importer()
```

Comment:
the import/export tests look very similar. Could you maybe make a base class (or method?) for them and just call them with the specification for each provider?